src/Microsoft.DotNet.Build.CloudTestTasks/DownloadFromAzure.cs

```diff
@@ -38,9 +38,20 @@ public sealed class DownloadFromAzure : AzureConnectionStringBuildTask
 
         public int MaxClients { get; set; } = 8;
 
+        [Output]
+        public ITaskItem[] PathTooLongBlobs
+        {
+            get
+            {
+                return pathTooLongEntries.ToArray();
```

Comment:
I expect this works but it may be confusing for later readers, generally output items are assigned in the context of the task execution, and it doesn't save any lines of code.  Any special reason?

Reply:
Ah, I hadn't noticed that pattern, there's lots of individual styles going on in this repo :)

So help me understand, why can't you just have a line like:

```
 PathTooLongBlobs = pathTooLongEntries.ToArray();
```

Say, here: https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.CloudTestTasks/DownloadFromAzure.cs#L110 ?